### ISSUE TITLE
Always try to install because fact: vmwaretools_version => not installed

### ISSUE THREAD

**Issue by juaningan** [craigwatson/puppet-vmwaretools#72]
I'm running Ubuntu 10.04 and install 9.0.10-1481436. All ends well but try to install in every puppet run.
Some info:

# facter -p vmwaretools_version
not installed

# file /usr/bin/vmware-toolbox-cmd
/usr/bin/vmware-toolbox-cmd: ERROR: cannot open `/usr/bin/vmware-toolbox-cmd' (No such file or directory)

# initctl list | grep vmware
vmware-tools start/running

# lsmod | grep vm
vmhgfs                 52548  0 
vmxnet                 15823  0 
vmmemctl                6444  0 
vmci                   73796  2 vmhgfs,vsock
vmw_pvscsi             13195  0 
vmxnet3                27174  0

**Comment by craigwatson** [comment]
Can you post the output of `which vmware-toolbox-cmd`?

**Comment by juaningan** [comment]
no output, and: find / -name vmware-toolbox-cmd neither

**Comment by craigwatson** [comment]
Thanks, can you post the full Puppet output with `--debug`?

**Comment by juaningan** [comment]
https://gist.github.com/5f279f789297617c720b.git

**Comment by craigwatson** [comment]
Seems GitHub 404s with that link, can you check?

**Comment by juaningan** [comment]
ok i've fixed the link. [delete this message]

**Comment by craigwatson** [comment]
I can't see anything majorly wrong in the output, can you try installing the tools manually, as I don't have a 10.04 system to hand?

Incidentally, 10.04 is officially unsupported from April 30th this year - see [this link](https://en.wikipedia.org/wiki/List_of_Ubuntu_releases#Table_of_versions)

**Comment by juaningan** [comment]
Ok, I've installed manually with default values and same problem happens. Then I launched intercative mode and installer notifies me that open-vm-tools are installed. So checking the vmware-install.pl script I've noticed that check the following packages: "open-vm-dkms", "open-vm-source", "open-vm-toolbox", "open-vm-tools",  "open-vm-tools-dbg". And, of course, "open-vm-source" was yet installed. I purged it and now all the binaries are there and module works ok.
